### PR TITLE
[ROS2] grid_map_cv port (re-submission)

### PR DESCRIPTION
**Pull request by Marwan99**
related to #219.

grid_map_cv package linted and ported.

This PR also includes a fix to grid_map_core/CMakeLists.txt which could be found in 48c72e0. This fix is addressing the omission of an extra cmake config file and the omission of exporting the include directories.

### CODE REVIEW

**Comment by Marwan99** [comment]
Not sure why CI is not triggering.
@SteveMacenski Any suggestions?

**Comment by SteveMacenski** [comment]
https://app.circleci.com/pipelines/github/ANYbotics/grid_map the only jobs I see are master build jobs, no PR jobs. did you rebase off of master? Maybe its not up to date in your branch if its older than the last merged PR when it was branched off

File: grid_map_cv/include/grid_map_cv/GridMapCvConverter.hpp [hunk 1]
```diff
@@ -87,8 +94,8 @@ class GridMapCvConverter
     float maxImageValue;
     if (std::is_same<Type_, float>::value || std::is_same<Type_, double>::value) {
       maxImageValue = 1.0;
-    } else if (std::is_same<Type_, unsigned short>::value || std::is_same<Type_, unsigned char>::value) {
-      maxImageValue = (float)std::numeric_limits<Type_>::max();
+    } else if (std::is_same<Type_, uint16_t>::value || std::is_same<Type_, unsigned char>::value) {
```
Comment (SteveMacenski) [inline grid_map_cv/include/grid_map_cv/GridMapCvConverter.hpp:97]: Why change unsigned short?

**Comment by SteveMacenski** [comment]
I didnt know about this inpainting filter... seems worthwhile. I wonder what other specific filters we could expose that would be really useful (topographical filters I imagine to remove speckle noise, maybe even a color clusterer)

File: grid_map_cv/include/grid_map_cv/GridMapCvConverter.hpp (hunk 1 above)
Comment (Marwan99) [reply grid_map_cv/include/grid_map_cv/GridMapCvConverter.hpp:97]: Linter complained about it

**Comment by Marwan99** [comment]
> https://app.circleci.com/pipelines/github/ANYbotics/grid_map the only jobs I see are master build jobs, no PR jobs. did you rebase off of master? Maybe its not up to date in your branch if its older than the last merged PR when it was branched off

I didnt rebase at all as it was failing to auto merge and the conflicts were not actual conflicts (it was comparing the same thing, not sure what was happening). This is a completely new branch branched off ros2.

File: grid_map_cv/include/grid_map_cv/GridMapCvConverter.hpp (hunk 1 above)
Comment (SteveMacenski) [reply grid_map_cv/include/grid_map_cv/GridMapCvConverter.hpp:97]: OK

**Comment by SteveMacenski** [comment]
Try closing and making a new PR with a branch off this one. I just enabled building PRs based on branches on forks

**Pull request closed without merging**